operators/pkg/forge/resourcequota.go

```diff
@@ -70,3 +76,13 @@ func TenantResourceQuotaSpec(quota *clv1alpha2.TenantResourceQuota) corev1.Resou
 		InstancesCountKey:             *resource.NewQuantity(int64(quota.Instances), resource.DecimalSI),
 	}
 }
+
+// EnforceSandboxResQuota updates the sandbox resource quota.
+func EnforceSandboxResQuota() corev1.ResourceList {
```

Comment:
```suggestion
func SandboxResourceQuotaSpec() corev1.ResourceList {
```

Please, be consistent.